serverlessworkflow/modules/ROOT/pages/cloud/kubernetes-service-discovery.adoc

```diff
@@ -0,0 +1,212 @@
+= Kubernetes Service Discovery for Serverless Workflow Application Configuration
+:compat-mode!:
+// Metadata:
+:description: Explain what is and how the service discovery works
+:keywords: kogito, workflow, quarkus, serverless, service-discovery, enhanced-service-discovery
+// links
+:quarkus_issue_url: https://github.com/quarkusio/quarkus/issues/27457
+
+
+The Kubernetes Service Discovery allows you to have a static URI that defines a Kubernetes resource that will be used
+to perform HTTP requests. The resource defined in the URI will then be queried in the current Kubernetes cluster and
+translated to a valid URL. The service discovery feature works during the Application startup, it scans all Quarkus
+configuration in search of the following URI pattern. Thus, you should keep in mind that, if the startup time matters to you, then you should consider to use a static URL instead.
+The following sketch demonstrates how this URI is composed:
+
+[source,shell]
+----
+kubernetes:<group>/<version>/<kind>/<namespace>/<name>?<attributeName>=<attributeValue>
+\________/ \_____/ \_______/ \____/ \_________/ \____/ \______________________________/
+  scheme    group   version   kind   namespace   name   additional resource attributes
+            \____________________/                       \__________________________/
+                     GVK                                   Supported values:
```

Comment:
`GVK` => you removed the `Kubernetes` before ?

Reply:
I didn't